packages/mdctl-cli/tasks/study.js

```diff
@@ -233,15 +237,36 @@ class Study extends Task {
     
     Usage: 
       
-      mdctl study [command]       
+      mdctl study [command] --manifestObject     
```

Comment:
given that we are adding this option to the CLI too may be we can call it `manifest` instead of `manifestObject` to make it consistent with the `manifest` option in `mdctl env export|import` command

Reply:
hi @nahueld I had that one initially, but weirdly it was somehow causing issues when generating the final manifest, so I decided to adopt a different name to avoid this